vm/runtime.ts

```diff
@@ -2146,3 +2290,8 @@ var TN = J2ME.throwNegativeArraySizeException;
 
 var PE = J2ME.preempt;
 var PS = 0; // Preemption samples.
+
+var OBJ_HDR_SIZE = J2ME.Constants.OBJ_HDR_SIZE;
```

Comment:
These should be marked as constants for closure to inline.  For all TS code we should should just use the full name Constant.XXX so it gets inlined by TS (and includes a nice debug comment about what the constant is).